### PR TITLE
Development kaiserfile

### PR DESCRIPTION
**Pull request by davidsiaw**
## Context

This change enables using rspec in a barcelona env upped with kaiser.

## How to test
1. `kaiser up`
2. `kaiser login rspec`

### CODE REVIEW

**Comment by degicat** [comment]
@Resonious please review this

File: Kaiserfile [hunk 1]
```diff
@@ -26,11 +30,11 @@ vault_host = ENV['VAULT_HOST'] || "#{vault_app_name}.localhost.labs.degica.com"
...
-  -e GITHUB_ORGANIZATION='degica'
-  -e GITHUB_DEVELOPER_TEAM='developers'
-  -e GITHUB_ADMIN_TEAM='Admin developers'
-  -e VAULT_URL=http://#{vault_host}
-  -e VAULT_PATH_PREFIX=degica
+  -e GITHUB_ORGANIZATION='dojiko'
+  -e GITHUB_DEVELOPER_TEAM='dev'
+  -e GITHUB_ADMIN_TEAM='admin'
+  -e VAULT_URL=https://#{vault_host}
+  -e VAULT_PATH_PREFIX=dojiko
```
Comment (Resonious) [inline Kaiserfile]: I had to remove these changes to get the specs to actually pass.

**Comment by davidsiaw** [comment]
@Resonious remind me to give you access to my dojiko org for teh testing justice

**Comment by davidsiaw** [comment]
@Resonious i figured out what you were talking about. It should be able to pass now as I've stubbed out the consts the original tests use.

**Pull request merged**